stage-openmower/45-openocd/00-run-chroot.sh

```
@@ -0,0 +1,21 @@
+# Create directory for OpenOCD
+mkdir -p /opt
+cd /opt
+
+# Clone OpenOCD repository
+git clone https://github.com/raspberrypi/openocd.git --recursive
+
+# Bootstrap, configure, compile, and install OpenOCD
+cd openocd
+./bootstrap
+./configure --enable-ftdi --enable-sysfsgpio --enable-bcm2835gpio --enable-linuxgpiod
+make -j$(nproc)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)